### DISCUSSION TITLE
Revit 2019 - Lookup install

### DISCUSSION THREAD

**Discussion by maltezc** [jeremytammik/RevitLookup#195] [Q&A] [answered]
Hello, Where is the Revit lookup install for Revit 2019? I remember it existed a couple of years ago but the earliest one i see now is for revit 2021.

**Comment by chuongmep** [comment, marked as answer]
I think you can get this version released : 
https://github.com/jeremytammik/RevitLookup/releases/tag/2023.1.0-EOL

**Comment by Nice3point** [comment]
https://github.com/jeremytammik/RevitLookup/wiki/Versions
